client/post-editor/editor-ground-control/index.jsx

```diff
@@ -393,11 +310,17 @@ const EditorGroundControl = React.createClass( {
 									? <Gridicon icon="scheduled" size={ 18 } />
 									: <Gridicon icon="calendar" size={ 18 } />
 								}
+								<span className="editor-ground-control__time-button__label">
+									{ postUtils.isFutureDated( this.props.post )
+										? this.moment( this.props.post.date ).calendar()
+										: this.translate( 'Choose date' )
```

Comment:
Given naming of other buttons (e.g. "Submit for Review"), I'm led to think we should capitalize the first character of both words here.